docs/source/user_guide/model_understanding.ipynb

```diff
@@ -254,9 +252,7 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "scrolled": false
```

Comment:
Is this important? Does it affect the visual output?

Reply:
This is the output before the change
![image](https://user-images.githubusercontent.com/22552445/103950920-c1a15d00-510b-11eb-8d4e-cb667863f1b6.png)

And this is the output after
![image](https://user-images.githubusercontent.com/22552445/103950941-cb2ac500-510b-11eb-8565-e19602d92cef.png)

They seem the same to me. @freddyaboulton fyi